packages/block-editor/src/components/font-sizes/with-font-sizes.js

```diff
@@ -64,7 +64,7 @@ export default ( ...fontSizeNames ) => {
 
 					createSetFontSize( fontSizeAttributeName, customFontSizeAttributeName ) {
 						return ( fontSizeValue ) => {
-							const fontSizeObject = find( this.props.fontSizes, { size: fontSizeValue } );
+							const fontSizeObject = find( this.props.fontSizes, { size: Number( fontSizeValue ) } );
```

Comment:
Hi, what was the reason for this change? fontSizeValue should already be a number right?

Reply:
The value retrieved from the select is [`event.target.value`](https://github.com/WordPress/gutenberg/blob/090586e6273dbd637b167d87b37b977e32a5a23f/packages/components/src/select-control/index.js#L34) which is a string.